Dockerfile

```diff
@@ -28,12 +28,11 @@ COPY --link --from=xx / /
 RUN xx-go --wrap
 RUN --mount=type=cache,target=/root/.cache/go-build \
     --mount=type=cache,target=/go/pkg/mod \
-    go build -o bin/spire-controller-manager main.go
+    go build -ldflags '-s -w' -o bin/spire-controller-manager main.go
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-#FROM gcr.io/distroless/static:nonroot
-FROM gcr.io/distroless/base AS spire-controller-manager
+# See https://github.com/chainguard-images/images/tree/main/images/static
+# Used to run static binaries, includes tzdata and ca-certificates
+FROM cgr.dev/chainguard/static:latest AS spire-controller-manager
```

Comment:
What benefits do we bring in by switching to this image over the base distroless image?

Reply:
Static image has ca-certificates and tzdata patched on a daily basis.

Furthermore it is build on Wolfi-OS (Linux Undistro) which aims to be memory safe, having SBOMs by default. E.g. we could install ca-certificates and tzdata in a builder image and copy it here, using this image we don't need to as Chainguard is having daily builds for these images including security checks against sboms and CVEs.